ui/i18n/messages.json

```diff
@@ -365,5 +365,7 @@
   "workflowButton.dialog.start": "Start",
   "workflowButton.dialog.heading": "Start Process",
   "workflowButton.dialog.placeholder": "Choose process",
-  "workflowButton.tooltip": "Start Process"
+  "workflowButton.tooltip": "Start Process",
+  "favoritesToggleButton.confirm.add": "Are you sure you want to add this document to Favorites?",
+  "favoritesToggleButton.confirm.remove": "Are you sure you want to remove this document from Favorites?"
```

Comment:
Usually, we try to keep the keys in the JSON object sorted alphanumerically, so these labels should be closer to the others starting with "f".